css/components/content-types/recipe/recipe.css

```diff
@@ -12,3 +12,115 @@
 
   Style guide: components.content_types.recipe
 */
+
+/* TODO: some of this could possibly be components / partials but lets get something to work with that's close */
+
+.page-node-type-recipe .title-main-wrapper {
+  background-color: #ffffff;
+  margin: 0 auto;
+  max-width: 1200px;
+  padding: 2.369em;
+}
+.page-node-type-recipe .layout__region--middle {
+  display: flex;
+  display: -ms-flexbox;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  margin-bottom: 2.369em;
+}
+.page-node-type-recipe .layout__region--middle .field__label {
+  font-family: 'Open Sans', Verdana, sans-serif;
+  font-size: 1rem;
+  line-height: 1.5rem;
+  margin: 0;
+  font-weight: 700;
+}
+.page-node-type-recipe .layout__region--middle-first,
+.page-node-type-recipe .layout__region--middle-second {
+  flex: 0 0 calc(50% - 1rem);
+}
+
+/* Recipe meta */
+.page-node-type-recipe .label-items {
+  display: inline-block;
+  margin-right: 1.5em;
+  font-size: 0.889rem;
+}
+.page-node-type-recipe .layout__region--middle-second {
+  display: flex;
+  display: -ms-flexbox;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-self: center;
+}
+.page-node-type-recipe .layout__region--middle-second > .recipe-meta {
+  flex: 0 0 40%;
+  text-align: center;
+  margin-bottom: 1.1845em;
+}
+.page-node-type-recipe .layout__region--middle-second > .field--name-field-summary {
+  flex-basis: 90%;
+  text-align: center;
+}
+.page-node-type-recipe .layout__region--middle-second svg {
+  height: 47px;
+}
+
+/* Recipe instructions */
+.page-node-type-recipe .layout__region--bottom {
+  background-color: #F4F2E9;
+  display: flex;
+  display: -ms-flexbox;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+.page-node-type-recipe .layout__region--bottom > h2 {
+  flex: 0 0 100%;
+  font-family: 'Scope One', Georgia, serif;
+  font-size: 2.369rem;
+  line-height: 1.2;
+  text-align: center;
+  padding: 1.2em;
+  margin: 0;
+}
+.page-node-type-recipe .layout__region--bottom-first {
+  flex: 0 0 33%;
+  padding: 0 2.369em 2.369em 2.369em;
+}
+.page-node-type-recipe .layout__region--bottom-second {
+  flex: 0 0 67%;
+  padding: 0 2.369em 2.369em 2.369em;
+}
+.page-node-type-recipe .layout__region--bottom .field__label {
+  font-family: 'Scope One', Georgia, serif;
+  font-size: 1.77rem;
+  line-height: 1.2;
+  padding: 0 0 0.6rem 0;
+  margin: 0 0 1.5rem 0;
```

Comment:
Margins and paddings should be related to the elements font size so it should use ems instead rems just like in line 83.

Reply:
Got it and changing